exec/java-exec/src/main/java/org/apache/drill/exec/server/options/TypeValidators.java

```diff
@@ -141,25 +141,41 @@ public void validate(final OptionValue v, final OptionManager manager) {
 
   public static class BooleanValidator extends TypeValidator {
     public BooleanValidator(String name, boolean def) {
-      super(name, Kind.BOOLEAN, OptionValue.createBoolean(OptionType.SYSTEM, name, def));
+      this(name, def, false);
+    }
+
+    public BooleanValidator(String name, boolean def, boolean isAdminOption) {
```

Comment:
Must be auto-generated code... Perhaps change "isAdminOption" to just "option" as a Boolean validator would seem to want to work with any boolean option...